package/CHANGELOG

```diff
@@ -21,6 +21,7 @@ The rules for this file:
  * 2.5.0
 
 Fixes
+  * Fix chi1_selections incorrectly returning None (Issue #4108)
```

Comment:
On second reading: make it more specific eg "Fix chi1_selections() ignored atom names CG1 OG OG1 SG and incorrectly returned None for amino acids CYS, ILE, SER, THR, VAL (Issue #4108)"

This will tell people immediately if they need this release.